<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Service</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_receive">
         <property name="text">
          <string>　　接收：　　</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_send">
         <property name="text">
          <string>　　发送：　　</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="ReceieMsgDisplayText"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="SendMsgDisplayText"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>741</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QLabel" name="label_data09">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="FileDataEdit_04"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_data02">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="FileDataEdit_09"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_data01">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="FileDataEdit_01"/>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_data13">
         <property name="text">
          <string>13</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_data05">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_data04">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="FileDataEdit_02"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="FileDataEdit_05"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="FileDataEdit_03"/>
       </item>
       <item row="0" column="7">
        <widget class="QLineEdit" name="FileDataEdit_13"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="FileDataEdit_06"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="FileDataEdit_07"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="FileDataEdit_08"/>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="FileDataEdit_10"/>
       </item>
       <item row="2" column="5">
        <widget class="QLineEdit" name="FileDataEdit_11"/>
       </item>
       <item row="3" column="5">
        <widget class="QLineEdit" name="FileDataEdit_12"/>
       </item>
       <item row="1" column="7">
        <widget class="QLineEdit" name="FileDataEdit_14"/>
       </item>
       <item row="2" column="7">
        <widget class="QLineEdit" name="FileDataEdit_15"/>
       </item>
       <item row="3" column="7">
        <widget class="QLineEdit" name="FileDataEdit_16"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_data06">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_data10">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_data14">
         <property name="text">
          <string>14</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label_data15">
         <property name="text">
          <string>15</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="label_data16">
         <property name="text">
          <string>16</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_data07">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_data11">
         <property name="text">
          <string>11</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_data08">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_data12">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_data03">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ReadFileDataBtn">
         <property name="text">
          <string>获取数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WriteFileDataBtn">
         <property name="text">
          <string>写入数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
